client/dive-common/components/RunPipelineMenu.vue

```diff
@@ -69,13 +87,16 @@ export default defineComponent({
         });
         // Filter out unsupported pipelines based on subTypeList
         // measurement can only be operated on stereo subtypes
-        if (name === stereoPipelineMarker) {
-          if (props.subTypeList.length === props.subTypeList.filter((item) => item === 'stereo').length) {
+        if (props.subTypeList.length === props.subTypeList.filter((item) => item === 'stereo').length
```

Comment:
`Array.prototype.every` would be more readable here.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/every